source/features/edit-comments-faster.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'dom-chef';
-import select from 'select-dom';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
 import PencilIcon from 'octicon/pencil.svg';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
 function init(): void {
-	const menuItems = select.all('details .js-comment-edit-button:not(.rgh-edit-comment)');
+	observe('.current-user .js-comment-header-reaction-button:not(.rgh-edit-comment)', {
```

Comment:
> ```
> .current-user
> ```

This is a substantial change that I didn't notice. It might make sense, but also requires a change of the description.

I think it's still worth keeping it for all the comments for consistency though

Reply:
This needs to be undone first

Reply:
> I think it's still worth keeping it for all the comments for consistency though

I get what you mean know 🤦‍♂️ I chose this class by mistake because I was testing in this repo